include/envoy/http/header_map.h

```diff
@@ -293,6 +293,7 @@ class HeaderEntry {
   HEADER_FUNC(EnvoyRetryGrpcOn)                                                                    \
   HEADER_FUNC(EnvoyRetriableStatusCodes)                                                           \
   HEADER_FUNC(EnvoyRetriableHeaderNames)                                                           \
+  HEADER_FUNC(EnvoyOriginalMethod)                                                                 \
```

Comment:
Drive by: please don't add this to the O(1) header map, as it only needs to be set.

Also, this change will need documentation and release notes. Thank you!

Reply:
Thanks for pointing this out. Could you point out where you would like to see the documentation? I could not find any prior mention of headers in https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/grpc_json_transcoder_filter. Should I create a section there anyway?

Reply:
Yeah I think documenting this within the filter config docs would be fine for now. Thank you!